src/frontend/Komponenter/Behandling/Brev/Htmlfelter.tsx

```diff
@@ -23,11 +26,16 @@ const lagRaderForVedtak = (tilkjentYtelse?: TilkjentYtelse): string => {
         .map((andel: AndelTilkjentYtelse) => {
             const inntekt = formaterTallMedTusenSkille(andel.inntekt);
             const beløp = formaterTallMedTusenSkille(andel.beløp);
-            return `<tr><td style="${borderStyling}">${formaterNullableIsoDato(
-                andel.stønadFra
-            )} - ${formaterNullableIsoDato(
+            const samordningsfradag = formaterTallMedTusenSkille(andel.samordningsfradrag);
+            return `<tr>
+                    <td style="${borderStyling}">
+                        ${formaterNullableIsoDato(andel.stønadFra)} - ${formaterNullableIsoDato(
                 andel.stønadTil
-            )}</td><td style="${borderStyling}">${inntekt}</td><td style="${borderStyling}">${beløp}</td></tr>`;
+            )}
+                    </td>
+                    <td style="${borderStyling}">${inntekt}</td>
+                    <td style="${borderStyling}">${beløp}</td>
+                    <td style="${borderStyling}">${samordningsfradag}</td></tr>`;
```

Comment:
Vil vi vise denne kolonnen hvis den er 0 for alle perioder?

Reply:
Nei, utbedret i 4278c7b